config.py

```diff
@@ -0,0 +1,58 @@
+import datetime
```

Comment:
nit: Can we also convert tabs to (4) spaces here? Sublime has a really nice way of doing this with just a few clicks.